packages/babel-plugin-proposal-function-sent/test/fixtures/function-sent/with-other-plugins/input.js

```diff
@@ -0,0 +1,8 @@
+let t;
+const o = {
+  *s(){
+    t ||= 'value'; // former logical assignment - ✔ Transformed
+    function.sent;
+    t ||= 'value'; // latter logical assignment - ❌ Not transformed
```

Comment:
Since we have fixed this bug, could you update the comment here? Otherwise it would be very confusing since it is supposed to be transformed.